src/routes/index.svelte

```diff
@@ -1,7 +1,57 @@
 <script lang="ts">
+  import { onMount } from 'svelte';
   import Authenticate from '$lib/components/Authenticate/index.svelte';
+
+  import type { Unsplash } from './api/unsplash';
+
+  let unsplash: Unsplash = null;
+
+  async function fetchBackground(): Promise<Unsplash> {
+    const result = await fetch('http://localhost:3000/api/unsplash');
```

Comment:
At some point having a `DOMAIN_URL` environment variable would be great. So we can build absolute URLs with ease anytime.